benchmarks/mongo-es-speed/index.js

```diff
@@ -0,0 +1,152 @@
+import { spawn } from 'child_process';
+import rawFs from 'fs';
+import { MongoClient } from 'mongodb';
+import nyanProgress from 'nyan-progress';
+import path from 'path';
+import pify from 'pify';
```

Comment:
Is it (entire library) really required for single `writeFile`? (May be just declare local function that wraps `fs.writeFile` into *promise*?)